lib/src/protobuf/json.dart

```diff
@@ -5,6 +5,14 @@
 part of protobuf;
 
 Map<String, dynamic> _writeToJsonMap(_FieldSet fs) {
+  return _writeToJsonMapWithMapper(fs, (fi) => '${fi.tagNumber}');
+}
+
+Map<String, dynamic> _writeToProto3JsonMap(_FieldSet fs) {
+  return _writeToJsonMapWithMapper(fs, (fi) => fi.name);
+}
+
+Map<String, dynamic> _writeToJsonMapWithMapper(_FieldSet fs, Function mapper) {
```

Comment:
If this is used you should make a typedef for the mapper argument. Something like:
```
typedef _FieldNamer = String Function(FieldInfo fieldInfo);

...

	Map<String, dynamic> _writeToJsonMapWithMapper(_FieldSet fs, _FieldNamer mapper) 
```